cmd/kube-apiserver/app/options/validation_test.go

```diff
@@ -89,7 +92,11 @@ func TestClusterServiceIPRange(t *testing.T) {
 			expectErrors: true,
 			options:      makeOptionsWithCIDRs("10.0.0.0/16", "3000::/64"),
 		},
-
+		{
+			name:         "more than two entries",
+			expectErrors: true,
+			options:      makeOptionsWithCIDRs("10.0.0.0/16,244.0.0.0/16", "3000::/108"),
```

Comment:
was the combination of two options into a single string intentional? this looks sort of weird

Reply:
Yes it was intentional, the intent was to cover this condition: https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-apiserver/app/options/validation.go#L46